test/e2e/kubernetes/util/util.go

```diff
@@ -2,11 +2,24 @@ package util
 
 import (
 	"fmt"
+	"log"
 	"os/exec"
 	"strings"
+	"time"
 )
 
 // PrintCommand prints a command string
 func PrintCommand(cmd *exec.Cmd) {
 	fmt.Printf("\n$ %s\n", strings.Join(cmd.Args, " "))
 }
+
+// RunAndLogCommand logs the command with a timestamp when it's run, and duration at and
+func RunAndLogCommand(cmd *exec.Cmd) ([]byte, error) {
```

Comment:
@jackfrancis @CecileRobertMichon are you ok with this tiny refactor as part of this PR? If not I can split it into a separate PR

Reply:
before
```
STEP: Creating a nginx deployment

$ kubectl run ingress-nginx-kubernetes-eastus-5726-17635 -n default --image library/nginx:latest --overrides { "apiVersion": "extensions/v1beta1", "spec":{"template":{"spec": {"nodeSelector":{"beta.kubernetes.io/os":"linux"}}}}} --labels=app=ingress-nginx
```


after
```
STEP: Creating a nginx deployment
2018/06/22 16:38:14 $ kubectl run nginx-kubernetes-eastus-67793-62845 -n default --image library/nginx:latest --overrides { "apiVersion": "extensions/v1beta1", "spec":{"template":{"spec": {"nodeSelector":{"beta.kubernetes.io/os":"linux"}}}}}
2018/06/22 16:38:14 #### $ kubectl run nginx-kubernetes-eastus-67793-62845 -n default --image library/nginx:latest --overrides { "apiVersion": "extensions/v1beta1", "spec":{"template":{"spec": {"nodeSelector":{"beta.kubernetes.io/os":"linux"}}}}} completed in 286.941082ms
```

Reply:
Fine with me but I think there's a typo at the end of the comment